client/ayon_houdini/startup/scripts/OnCreated.py

```diff
@@ -0,0 +1,18 @@
+# Any code here will run on any node being created in Houdini
+# As such, preferably the code here should run fast to avoid slowing down node
+# creation. Note: It does not trigger on existing nodes for scene open nor on
+# node copy-pasting.
+from ayon_core.lib import env_value_to_bool, is_dev_mode_enabled
+from ayon_houdini.api import node_wrap
+
+
+# Allow easier development by automatic reloads
+# TODO: remove this
+if is_dev_mode_enabled():
```

Comment:
I don't agree with this, but I'm not responsible for houdini repository.

Do you know it will "reload" the module only for newly imported files (only plugins), and may break other imports because of module pointers cleanup (don't know if still happens in Py3).

Reply:
Correct, this should not be in live code - this is just there for my own dev. Ignore it for now, the "remove this" todo completely applies.